build/release/release.sh

```diff
@@ -101,6 +101,9 @@ upload_nexus_staging() {
     -s "${KYUUBI_DIR}/build/release/asf-settings.xml" \
     -pl extensions/spark/kyuubi-extension-spark-3-2 -am
   ${KYUUBI_DIR}/build/mvn clean deploy -DskipTests -Papache-release,flink-provided,spark-provided,hive-provided,spark-3.3 \
+    -s "${KYUUBI_DIR}/build/release/asf-settings.xml" \
+    -pl extensions/spark/kyuubi-extension-spark-3-3 -am
+  ${KYUUBI_DIR}/build/mvn clean deploy -DskipTests -Papache-release,flink-provided,spark-provided,hive-provided,spark-3.4 \
```

Comment:
Kyuubi Spark 3.4 extension?

Reply:
Kyuubi compiles against Spark 3.4 now, the last one contains Kyuubi Spark 3.4 extension

Reply:
Got it, the Kyuubi Spark 3.4 extension will be active by default.